cmd/vic-ui/main.go

```diff
@@ -0,0 +1,98 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	log "github.com/Sirupsen/logrus"
+
+	"github.com/urfave/cli"
+	"github.com/vmware/vic/cmd/vic-ui/ui"
+	"github.com/vmware/vic/pkg/errors"
+)
+
+var (
+	Version  string
+	BuildID  string
+	CommitID string
+)
+
+const (
+	LogFile = "vic-ui.log"
+)
+
+func main() {
+	app := cli.NewApp()
+
+	app.Name = filepath.Base(os.Args[0])
+	app.Usage = "Install/remove VIC UI plugin"
+	app.EnableBashCompletion = true
+
+	ui := ui.NewUI()
+	app.Commands = []cli.Command{
+		{
+			Name:   "install",
+			Usage:  "Install UI plugin",
+			Action: ui.Install,
+			Flags:  ui.Flags(),
+		},
+		{
+			Name:   "remove",
+			Usage:  "Remove UI plugin",
+			Action: ui.Remove,
+			Flags:  ui.Flags(),
```

Comment:
Maybe use different set of flags with install? that will make user a little bit easier. But I know it's internal user only, so up to you.